streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -99,11 +100,18 @@
 
         // only for active restoring tasks (for standby changelog it is null)
         // NOTE we do not book keep the current offset since we leverage state manager as its source of truth
-        private Long restoreEndOffset;
 
-        // only for standby tasks that use source topics as changelogs (for active it is null);
-        // if it is not on source topics it is also null
-        private Long restoreLimitOffset;
```

Comment:
Following the suggestion from KAFKA-9113 PR, by @ableegoldman we consolidate the limit-offset with end-offset.